Extensions/Kayenta/Src/vss-extension.json

```diff
@@ -0,0 +1,177 @@
+{
+    "manifestVersion": 1,
+    "id": "vss-services-kayentadeploymentgate",
+    "name": "Kayenta Canary Judge",
+    "publisher": "ms-vscs-rm",
+    "version": "0.154.2",
+    "public": false,
+    "description": "Integrate Kayenta Canary Judge with Azure Pipelines",
+    "categories": [
+      "Azure Pipelines"
+    ],
+    "Tags": [
+      "Kayenta",
+      "Automated Canary Analysis",
+      "Canary",
+      "Release",
+      "Release Gates",
+      "DevOps",
+      "Pipelines"
+    ],
+    "targets": [
+      {
+        "id": "Microsoft.VisualStudio.Services.Cloud"
+      },
+      {
+
+        "id": "Microsoft.TeamFoundation.Server"
```

Comment:
Remove extra empty lines.
Check with Aseem once if we want to enable it for OnPrem or not.